modules/shared_data/src/main/resources/application.conf

```diff
@@ -41,7 +41,7 @@ rewards {
       liquidity-pools-config = [
         {
           start-epoch = 1
-          minimum-shares = 0
+          minimum-shares = 1
```

Comment:
@joaoCarvalho1000 after that change at least one share is required for node validator for receiving rewards